cmd/memberagent/main.go

```diff
@@ -72,7 +73,7 @@ func main() {
 	}
 	tokenFilePath := os.Getenv("CONFIG_PATH")
 
-	if tokenFilePath == "" {
+	if !*useCaAuth && tokenFilePath == "" {
 		klog.ErrorS(errors.New("hub token file path cannot be empty"), "error has occurred retrieving CONFIG_PATH")
```

Comment:
Error message no longer reflects that neither the token file path nor useCaAuth bundle is set to be used.

Reply:
Thanks for catching it up. Update the error msg to reflect the CA auth part.